core/templates/services/svg-sanitizer.service.ts

```diff
@@ -94,15 +106,23 @@ export class SvgSanitizerService {
 
   private _getInvalidSvgTagsAndAttrs(
       svg: Document): {tags: string[], attrs: string[]} {
-    let invalidTags = [];
-    let invalidAttrs = [];
-    let allowedTags = Object.keys(constants.SVG_ATTRS_WHITELIST);
+    let invalidTags: string[] = [];
+    let invalidAttrs: string[] = [];
+
+    // This throws "The type 'readonly ["about", "alignment-baseline, ... 86
+    // more ..., "writing-mode"]' is 'readonly' and cannot be assigned to the
+    // mutable type 'string[]'." We need to suppress this error because of
```

Comment:
This seems odd and needs a proper resolution. Is the second argument in svgAttrsWhitelist supposed to be readonly or not?

Reply:
We can't assign a readonly value to const or a variable, this is also being tracked by a issue here https://github.com/Microsoft/TypeScript/issues/13347 from so long with no satisfactory solution. I also tried making the type readonly but that doesn't works. So I think this should be ignored, the type seems correct apart from `readonly` thing. wdyt ?
![Screenshot from 2021-07-18 17-41-32](https://user-images.githubusercontent.com/60519303/126066731-56cc4ca5-9d73-4a07-95b9-0409dd60cbd9.png)
![image](https://user-images.githubusercontent.com/60519303/126066756-b1973214-1542-45b4-9323-cf3f3ab05e3c.png)

Reply:
ts-ignore should be your last resort. Can you cast the readonly value to the proper type instead?

The issue you linked seems to be the other way round, it says an assignment is allowed whereas it should be forbidden. So I'm not sure how it relates to this case.

Reply:
Updated!